### DISCUSSION TITLE
How to get flux residuals’ error?

### DISCUSSION THREAD

**Discussion by TheCXT** [phoebe-project/phoebe2#557] [Questions]
![flux residuals](https://user-images.githubusercontent.com/75238101/140602353-d76e8809-7f80-4cfc-be11-3959914bbf63.png)
![code](https://user-images.githubusercontent.com/75238101/140602391-d333ff5d-0821-4392-bb2f-626c5f89237e.jpg)

Hello，
I have question. I've calculated and got the flux residuals, and I want to get flux residuals’ error, what do I do?

**Comment by kecnry** [comment]
See #534 and #556.  The plotted errors are the same as the errors you input with the flux observations and can be accessed from the parameter in the dataset (or from the original array).
